scanner/src/main/kotlin/provenance/ProvenanceDownloader.kt

```diff
@@ -66,7 +66,19 @@ fun interface ProvenanceDownloader {
         nestedProvenance.subRepositories.forEach { (path, provenance) ->
             val tempDir = download(provenance)
             val targetDir = root.resolve(path).apply { parentFile.safeMkdirs() }
-            tempDir.toPath().moveTo(targetDir.toPath(), StandardCopyOption.ATOMIC_MOVE)
+
```

Comment:
Unfortunately, there is no commit message body to explain what the intention of this code is, what it does, and why it's needed. Specifically, why is it not enough to only swap the priority values of the VCS implementations?